tools/CouchDB_Import_MRI.php

```diff
@@ -122,11 +190,33 @@ function UpdateCandidateDocs($data, $ScanTypes) {
     }
 
     function run() {
-        $ScanTypes = $this->SQLDB->pselect("SELECT DISTINCT pf.ParameterTypeID, pf.Value as ScanType from parameter_type pt JOIN parameter_file pf USING (ParameterTypeID) WHERE pt.Name='selected' AND COALESCE(pf.Value, '') <> ''", array());
+        $ScanTypes = $this->SQLDB->pselect("SELECT DISTINCT pf.ParameterTypeID,
+                          pf.Value as ScanType
+                     FROM parameter_type pt
+                     JOIN parameter_file pf
+                     USING (ParameterTypeID)
+                     WHERE pt.Name='selected' AND COALESCE(pf.Value, '') <> ''", array());
         $this->UpdateDataDict($ScanTypes);
         $query = $this->_generateCandidatesQuery($ScanTypes);
         $CandidateData = $this->SQLDB->pselect($query, array());
         $this->UpdateCandidateDocs($CandidateData, $ScanTypes);
+        $CandidateData = $this->SQLDB->pselect($query, array());//print_r($CandidateData);
+        foreach ($CandidateData as &$row) {
+            foreach ($ScanTypes as $scanType) {
```

Comment:
$ScanTypes is not defined. 

Should run this first:
$ScanTypes = $this->SQLDB->pselect("SELECT DISTINCT pf.ParameterTypeID,
                          pf.Value as ScanType
                     FROM parameter_type pt
                     JOIN parameter_file pf
                     USING (ParameterTypeID)
                     WHERE pt.Name='selected' AND COALESCE(pf.Value, '') <> ''", array());